doc/rados/operations/balancer.rst

```diff
@@ -93,43 +102,41 @@ The numeric pool IDs can be gotten with the :command:`ceph osd pool ls detail` c
 Modes
 -----
 
-There are currently two supported balancer modes:
+There are two supported balancer modes:
 
-#. **crush-compat**.  The CRUSH compat mode uses the compat weight-set
-   feature (introduced in Luminous) to manage an alternative set of
-   weights for devices in the CRUSH hierarchy.  The normal weights
-   should remain set to the size of the device to reflect the target
-   amount of data that we want to store on the device.  The balancer
-   then optimizes the weight-set values, adjusting them up or down in
-   small increments, in order to achieve a distribution that matches
-   the target distribution as closely as possible.  (Because PG
-   placement is a pseudorandom process, there is a natural amount of
-   variation in the placement; by optimizing the weights we
-   counter-act that natural variation.)
+#. **crush-compat**. The CRUSH compat mode uses the compat weight-set feature
```

Comment:
This mode uses

Reply:
Oh, alright. This change was 100 percent my prejudice for sentences that can be quoted out of context, but I bow to the insight of your fresh eyes.

Reply:
There's value in that sometimes.  Here " c r u s h - c o m p a t " is a literal and the CRUSH acronym doesn't really fit, that's really what I was going for.